src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,376 @@
+            
+            self.assertEqualToDeviceOrPreloadedCondtionalParam('longitude', \
+             device['installationParam'], reg_conditional_device_data\
+             ['installationParam'], cbsd_record[0]['installationParam'])
+            
+            self.assertEqualToDeviceOrPreloadedCondtionalParam('height', \
+             device['installationParam'], reg_conditional_device_data\
+             ['installationParam'], cbsd_record[0]['installationParam'])
+
+            self.assertEqualToDeviceOrPreloadedCondtionalParam('heightType', \
+             device['installationParam'], reg_conditional_device_data\
+             ['installationParam'], cbsd_record[0]['installationParam'])
+            
+            self.assertEqualToDeviceOrPreloadedCondtionalParam('antennaGain', \
+             device['installationParam'], reg_conditional_device_data\
+             ['installationParam'], cbsd_record[0]['installationParam'])                   
+            
+             # parameters should exist in record if exist in device,\        
+            self.assertEqualToIfExistDeviceOrPreloadedCondtionalParam('indoorDeployment', \
+                device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'])
+            #  parameters should equal to device or to default value            
+            self.assertEqualToDeviceOrPreloadedConditionalParamOrDefaultValue('antennaAzimuth', \
+                device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'], 0)
+                
+            self.assertEqualToDeviceOrPreloadedConditionalParamOrDefaultValue('antennaBeamwidth', \
+               device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam'], 360)
+            
+            # if callSign exist, it should have the same value as registered
+            if 'callSign' in cbsd_record[0]:
+                self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])      
+            max_eirp_by_MHz = 37;     
+            if 'eirpCapability' in cbsd_record[0]:
+              max_eirp_by_MHz = cbsd_record[0]['eirpCapability'] - 10
+            # optional parameters if exists in the dump record should be equal to the device's value
+            self.assertEqualToDeviceOrPreloadedConditionalOptionalParam('antennaModel', \
+               device['installationParam'], reg_conditional_device_data['installationParam'],\
+                 cbsd_record[0]['installationParam']) 
+                  
+            self.assertEqualToDeviceOrPreloadedConditionalOptionalParam('eirpCapability', \
+               device, reg_conditional_device_data,\
+                 cbsd_record[0])
+
+            self.assertEqualToDeviceOrPreloadedConditionalOptionalParam('horizontalAccuracy', \
```

Comment:
I deleted the checks of optional feilds, to be coherent with other test cases, but I had added them because I thought that even if they are optional we should check that the dump doesn't contain some strange/incorrect values.

Reply:
I thought the same but was told to do otherwise in the SAS Admins group (you can check with Christophe) :/